test/disconnector_test.dart

```diff
@@ -84,7 +84,7 @@ void main() {
     var sinkController = new StreamController();
     var disconnector = new Disconnector();
     var sink = new _CloseCompleterSink(sinkController.sink);
-    var channel = new StreamChannel.withGuarantees(
+    channel = new StreamChannel.withGuarantees(
```

Comment:
Because `var channel` is already defined on line 17, so the analyzer complains about masking the other declaration here.